packages/esm-ward-app/src/ward-workspace/admission-requests.workspace.tsx

```diff
@@ -29,7 +27,7 @@ const AdmissionRequestsWorkspace: React.FC<DefaultWorkspaceProps> = () => {
 
   return (
     <div className={styles.admissionRequestsWorkspace}>
-      {admissionRequests.map((admissionRequest, indx) => (
+      {inpatientRequests.map((admissionRequest, indx) => (
         <AdmissionRequestCard key={indx} patient={admissionRequest.patient} />
```

Comment:
Is there something in the response we could use as a key here?

Reply:
Or just like `inpatient-request-workspace-card-${indx}`.

@vasharma05 sorry we didn't catch this in the original PR with this change, but you can't just use an integer as a React key; it is very easy to wind up with duplicate keys that way. Duplicate keys can cause really bizarre and confusing bugs.

Reply:
```suggestion
        <AdmissionRequestCard key={`inpatient-request-workspace-card-${indx}`} patient={admissionRequest.patient} />
```